src/net_protocol.rs

```diff
@@ -149,6 +152,14 @@ impl<S: crate::store::Store> Blobs<S> {
         &self.store
     }
 
```

Comment:
I think down in this file there are a bunch of method which are public, only to make RPC work, would be good to check which of those you can now make private

Reply:
Making all of this private would be a bit of effort.